exocompute/version-1/permission-group-PRIVATE_ENDPOINTS.json

```diff
@@ -0,0 +1,79 @@
+[
```

Comment:
There is an `exocompute/version-2` directory. Does that not need this permission group?

Reply:
We don't need to put it in the version-2 folder as this permission group does not have version 2. That is how the permission parity test works currently.